connectors/uplynk/src/main/java/com/theoplayer/android/connector/uplynk/network/AssetInfoResponse.kt

```diff
@@ -59,7 +59,7 @@ data class AssetInfoResponse(
      *      <li>7: Not Rated
      * </ul>
      *
-     * @return The {@link VerizonMediaAssetTvRating}. (<b>NonNull</b>)
+     * @return The {@link UplynkMediaAssetTvRating}. (<b>NonNull</b>)
```

Comment:
This doesn't link to anything... Just remove it, it's already fully described by the list above.